### PR TITLE
Add Svelte

### PR DESCRIPTION
**Pull request by haradan**
Use vite_rails to add Svelte components to any Rails view

### CODE REVIEW

**Review by plural — COMMENTED** [review]
Would you please split out your demo template and your route changes and keep this just to adding the deps and putting the supporting files in place?

File: config/initializers/content_security_policy.rb [hunk 1]
```diff
@@ -11,7 +11,16 @@
 #     policy.img_src     :self, :https, :data
 #     policy.object_src  :none
 #     policy.script_src  :self, :https
+    # Allow @vite/client to hot reload javascript changes in development
```
[the remark this answers is not in the document]
Comment (haradan) [reply config/initializers/content_security_policy.rb]: Done

**Comment by haradan** [comment]
> Would you please split out your demo template and your route changes and keep this just to adding the deps and putting the supporting files in place?

I've done that now

**Review by plural — APPROVED** [review]

File: Procfile.dev
[the remark this answers is not in the document]
Comment (haradan) [reply Procfile.dev:1]: I think it tells Rails to run the Vite development server when the app starts in development mode:

https://railsnotes.xyz/blog/procfile-bin-dev-rails7

**Pull request merged**